python/cudf/cudf/core/multiindex.py

```diff
@@ -1474,7 +1474,7 @@ def _clean_nulls_from_index(self):
 
     @_cudf_nvtx_annotate
     def memory_usage(self, deep=False):
-        usage = sum(super().memory_usage(deep=deep).values())
+        usage = sum([col.memory_usage for col in self._data.columns])
```

Comment:
You can use a generator expression directly.
```suggestion
        usage = sum(col.memory_usage for col in self._data.columns)
```